en/reference/annotations-mapping.rst

```diff
@@ -74,33 +74,35 @@ Minimal example:
 
 .. code-block:: php
 
-   <?php
```

Comment:
Commenting once for many instances: some code blocks have an opening tag, others don't. Do you mind taking a look at that as well?

Reply:
my logic was: when its code that would work on its own (e.g. a class) i put the opening tag, but when its just a fragment like a property declaration, i remove it. does that make sense? or would you simply remove it everywhere?

Reply:
oh, but around here, i have been super not consistent with my own plan. i think i will just remove all of the `<?php`

Reply:
Your reasoning makes perfect sense 👍